src/components/App.tsx

```diff
@@ -470,6 +469,8 @@ export class App extends React.Component<any, AppState> {
       });
     }
 
+    globalSceneState.addCallback(this.handleSceneCallback);
```

Comment:
Can you cleanup when the component unmounts? This is not needed right now since we never unmount but it's good hygiene to.